oauth2/src/test/java/com/linecorp/armeria/client/auth/oauth2/OAuth2ClientCredentialsGrantTest.java

```diff
@@ -190,49 +190,46 @@ void testConcurrent() throws Exception {
         final List<AggregatedHttpResponse> responses1 =
                 getConcurrently(client, "/resource-read-write/", count).join();
         validateResponses(responses1, HttpStatus.OK);
-        verify(grant, times(1)).obtainAccessToken(any());
-        verify(grant, times(0)).refreshAccessToken(any(), any());
+        assertThat(newTokenCounter.get()).isOne();
 
         final List<AggregatedHttpResponse> responses2 =
                 getConcurrently(client, "/resource-read/", count).join();
         validateResponses(responses2, HttpStatus.OK);
-        verify(grant, times(1)).obtainAccessToken(any());
-        verify(grant, times(0)).refreshAccessToken(any(), any());
+        assertThat(newTokenCounter.get()).isOne();
 
         final List<AggregatedHttpResponse> responses3 =
                 getConcurrently(client, "/resource-read-write-update/", count).join();
         validateResponses(responses3, HttpStatus.FORBIDDEN);
-        verify(grant, times(1)).obtainAccessToken(any());
-        verify(grant, times(0)).refreshAccessToken(any(), any());
+        assertThat(newTokenCounter.get()).isOne();
```

Comment:
```
    DefaultOAuth2AuthorizationGrant(..) {
        tokenLoader = tokenLoader();
    }

FAILED: wanted but not invoked obtainAccessToken, only invoked getAccessToken();
```
We make tokenLoader in constructor, then mockito can't verify `obtainAccessToken, refreshAccessToken` invocation well.

So I replace it with newTokenCounter :)